### ISSUE TITLE
Audit #1 – Share a single file – Accordion Focus not visible

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4791]
Connected to Audit 1, see #4300 
1.5 General Overview when logged in, URL: https://ocis-a11y.owncloud.works/#/files/list

# Issue
- Accordion triggers can be operated with the keyboard, but focus is not visible

# Remediation
Apply focus style, for example the one used in buttons

**Comment by JammingBen** [comment]
Was fixed in one of the latest ODS releases.
